st2common/st2common/operators.py

```diff
@@ -22,10 +22,24 @@
     'get_allowed_operators'
 ]
 
+
+def get_allowed_operators():
+    return operators
+
+
+def get_operator(op):
+    op = op.lower()
+    if op in operators:
+        return operators[op]
+    else:
+        raise Exception('Invalid operator: ' + op)
+
 # Operation implementations
 
 
 def equals(value, criteria_pattern):
+    if criteria_pattern is None:
```

Comment:
I was against like doing something like that in the past (I prefer being explicit and have strong types where possible - if it won't break in the rule not, it might break somewhere down the pipeline though), but oh well, lets say I'm OK with it now.